packages/aws-appsync-subscription-link/src/realtime-subscription-handshake-link.ts

```diff
@@ -385,6 +390,21 @@ export class AppSyncRealTimeSubscriptionHandshakeLink extends ApolloLink {
           const headerQs = Buffer.from(headerString).toString("base64");
 
           const payloadQs = Buffer.from(payloadString).toString("base64");
+
+          let discoverableEndpoint = appSyncGraphqlEndpoint;
+
+          if (this.isCustomDomain(discoverableEndpoint)) {
+					    discoverableEndpoint = discoverableEndpoint.concat(
+							    customDomainPath
```

Comment:
@manueliglesias @david-mcafee This change is breaking our use case in which we point to localhost in development but the endpoint for subscriptions is still `/graphql` and not `/realtime`. It doesn't seem to make sense to me that the library should modify the provided custom endpoint but maybe I missed something in this PR.

Any suggestions?